### PR TITLE
add labels to issue templates

### PR DESCRIPTION
**Pull request by OskarStark**
see https://blog.github.com/changelog/2018-12-05-issue-template-automation-improvements/

### CODE REVIEW

File: project/.github/ISSUE_TEMPLATE/Bug.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 name: 🐞 Bug Report
 about: Something is broken? 🔨
+title: ''
```
Comment (kunicmarko20) [inline project/.github/ISSUE_TEMPLATE/Bug.md]: what is the point of title if it is empty?

File: project/.github/ISSUE_TEMPLATE/Bug.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+labels: bug, unconfirmed
+assignees: ''
```
Comment (kunicmarko20) [inline project/.github/ISSUE_TEMPLATE/Bug.md]: same question

File: project/.github/ISSUE_TEMPLATE/Bug.md (hunk 1 above)
Comment (OskarStark) [reply project/.github/ISSUE_TEMPLATE/Bug.md]: Its generated by the github ui in this way. You . can add a default title like:
`[BUG] `, but imo its to verbose when we have the bug label

Comment (greg0ire) [reply project/.github/ISSUE_TEMPLATE/Bug.md]: You mean it breaks when it is missing?

Comment (greg0ire) [reply project/.github/ISSUE_TEMPLATE/Bug.md]: Obviously not since we already use it that way, I'm dumb… so what's the point?

Comment (Soullivaneuh) [reply project/.github/ISSUE_TEMPLATE/Bug.md]: I agree with @greg0ire. If the option is not mandatory I would recommend to not add it if not used.

**Pull request merged**